targets.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <Choose>
+    <When  Condition="'$(VisualStudioVersion)' == '16.0'">
+      <PropertyGroup>
+        <AppTargetFramework>netcoreapp3.1</AppTargetFramework>
+        <TestsTargetFrameworks>net462;netcoreapp2.1;netcoreapp3.1</TestsTargetFrameworks>
+        <LibTargetFrameworks>net462;netstandard2.0;netstandard2.1</LibTargetFrameworks>
+        <LibxTargetFrameworks>net462;netcoreapp2.0;netstandard2.1</LibxTargetFrameworks>
```

Comment:
When will LibxTargetFrameworks be used?

Reply:
Its used in Libraries/Opc.Ua.Client.ComplexTypes/Opc.Ua.Client.ComplexTypes.csproj

Reply:
indeed